posthog/request.py

```
@@ -68,7 +68,17 @@ def _process_response(
     log = logging.getLogger("posthog")
     if res.status_code == 200:
         log.debug(success_message)
-        return res.json() if return_json else res
+        response = res.json() if return_json else res
+        # Handle quota limited decide responses by raising a specific error
+        if (
+            isinstance(response, dict)
+            and "quotaLimited" in response
+            and isinstance(response["quotaLimited"], list)
+            and "feature_flags" in response["quotaLimited"]
```

nit: I'm assuming this is to cover the `/decide` case, correct? Can / should we avoid the special condition for `feature_flags` and instead log any quota limited services, maybe raising an error for any of them (or, keep a list of services that should throw an error if quota limited)

---

oh yeah, good call. I think for now I explicitly want to limit feature flag events for now (since I don't know how other services are handling quota limits in the SDK).

---

I'll make a comment to that end though.